pg_bm25/src/index_access/delete.rs

```diff
@@ -9,18 +8,21 @@ pub extern "C" fn ambulkdelete(
     callback: pg_sys::IndexBulkDeleteCallback,
     callback_state: *mut ::std::os::raw::c_void,
 ) -> *mut pg_sys::IndexBulkDeleteResult {
-    let mut stats_binding = stats;
+    let info = unsafe { PgBox::from_pg(info) };
+    let mut stats = unsafe { PgBox::from_pg(stats) };
```

Comment:
Can you help me understand why you chose to `PgBox` these values? It seems like they're already managed by Postgres.

Reply:
For `stats` I think it's because we needed to modify it in `buld_delete`, and in my view it's better to have as little `unsafe` pointer dereferences in the codebase as possible. Sure, there's a memory impact to boxing, and if that becomes a concern then we can remove it. At this stage of the project I personally favor ease of working with the codebase (e.g eliminating as many ugly `unsafe` sections as possible) vs. deep performance optimizations. 
But, in case you guys think otherwise, the boxing should be removed yes. 

For `info`, can't really remember anymore, I think that was already boxed even before? Not sure if I changed that tbh...